decentraland/AppNavigator.js

```diff
@@ -23,6 +23,7 @@ import { responsiveHeight } from "react-native-responsive-dimensions";
 const defaultNavigationOptions = {
   headerStyle: {
```

Comment:
Does headerStyle here work the same as `header.style`? Or do we use this as `header.style` elsewhere? https://github.com/tasitlabs/tasit/pull/293/files/c70f2340aa8daa8a4ab7afedfad3b6e54ca842ab..28f362a255a5432738b009af4a34d9935364492e#r275882094

Reply:
`headerStyle` works, it is used on the [docs](https://reactnavigation.org/docs/en/headers.html#adjusting-header-styles) and those color changes that we have done was made in there.

Reply:
Great, thanks. Resolving this.

Reply:
Since we can have a `navigationOptions` per screen I think that allows us to customize styles too.

Reply:
Oh, cool. Good to know